src/libYARP_dev/include/yarp/dev/ILocalization2D.h

```diff
@@ -12,10 +12,18 @@
 #include <yarp/os/Vocab.h>
 #include <yarp/os/Log.h>
 #include <yarp/dev/Map2DLocation.h>
+#include <vector>
 
 namespace yarp {
     namespace dev {
         class ILocalization2D;
+
+        enum LocalizationStatusEnum
```

Comment:
Since it a new "enum", what about using `enum class` instead? (See #1760)

Reply:
cannot use enum class because bugged in gcc when using with extern (see some issue around)
fixed in ubuntu 18.04